docs/modules/reference/pages/daemons/introduction.adoc

```diff
@@ -0,0 +1,50 @@
+= List of daemons
+
+.List of daemons
+[options="header"]
+[cols="1,3,2"]
```

Comment:
How many columns in this table? Three numbers here indicates there should be three columns.
```suggestion
[cols="1,3"]
```